apps/system/js/lockscreen_window_manager.js

```diff
@@ -89,12 +89,55 @@
     this.startEventListeners();
     this.startObserveSettings();
     this.initElements();
-    this.initWindow();
+    this._fetchFTUStatus().then((status) => {
```

Comment:
I tend to think `this._fetchFTUStatus()` should simply return ONE promise resolve to boolean since the caller doesn't need the detail, as evidenced below.